lib/mods/replace-constructors.js

```diff
@@ -27,15 +39,21 @@ function getGlobalNewExpressions(j, root) {
 };
 
 /**
- * Gets the original name of the identifier param
+ * Adds an import for the module at the top of the file
  *
- * @param {String} name - the name of the identifier
- *
- * @return {String} the name of the original version
+ * @param {String} module - the name of the module to import from
+ * @param {String} original - the name of the original to get
+ * @param {Collection} root - the root Collection
  */
-function getOriginal(name) {
+function addImport(module, original, root) {
   // TODO(jackbsteinberg): implement
-  return 'original_' + name;
+  // if an import of original from module exists, noop
+
+  // if an import from module exists without importing original,
+  // add original to the import
+
+  // if no import from module exists, add a new import
+  // of original, from module
```

Comment:
Be sure to add tests for all three of these cases. Probably involving multiple test files!